Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -393,6 +393,8 @@ extension ActivityStreamPanel {
             return topSitesManager.content.isEmpty ? 0 : 1
         case .highlights:
             return self.highlights.count
+        case .pocket:
+            return self.pocketStories.count
```

Comment:
Instead here, return the number of items for a row. instead of the total count. (as long as pocket stories isnt empty)